### ISSUE TITLE
Update pyxform command

### ISSUE THREAD

**Issue by esthermmoturi** [medic/cht-docs#1558]
On the [local setup page](https://docs.communityhealthtoolkit.org/building/local-setup/)  in the [pyxform section](https://docs.communityhealthtoolkit.org/building/local-setup/#pyxform):

1.  Indicate that an environment has to be created and activated (for python3)
2.  Replace with this command `sudo python3 -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic`
3. Add `pip install pyxform`

**Comment by mrjones-plip** [comment]
> 3. Add pip install pyxform

you don't need to do this step as step 2 does it for you but uses Medic's specific version that is needed.

**Comment by esthermmoturi** [comment]
For some reason, before running `pip install pyxform` I was not getting any output for `pip show pyxform`. Is there any other way to confirm that pyxform is installed?

**Comment by sugat009** [comment]
You could try this:
```bash
pip freeze | grep pyxform
```

**Comment by mrjones-plip** [comment]
what we're actually doing is installing the `xls2xform-medic` binary.  For me this got installed here:

```
which xls2xform-medic
/home/mrjones/.local/bin/xls2xform-medic
```

And since `~/.local/bin` is in my path, I can just test by calling `xls2xform-medic` and see the error output that  it's missing arguments:

```
xls2xform-medic         
usage: xls2xform-medic [-h] [--json] [--skip_validate] path_to_XLSForm output_path
xls2xform-medic: error: the following arguments are required: path_to_XLSForm, output_path
```

**Comment by esthermmoturi** [comment]
Thanks @mrjones-plip and @sugat009 , I get it.

**Comment by esthermmoturi** [comment]
Closed by #1571
